src/Command/Member/AbstractMemberCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace SilverLeague\Console\Command\Member;
+
+use SilverLeague\Console\Command\SilverStripeCommand;
+use SilverStripe\Security\Member;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Provides a base for member related commands, lookups etc
+ *
+ * @package silverstripe-console
+ * @author  Robbie Averill <robbie@averill.co.nz>
+ */
+abstract class AbstractMemberCommand extends SilverStripeCommand
```

Comment:
If I'm extremely picky... This class doesn't provide any abstract methods, so is it actually an abstract class?

Reply:
It's abstract because it shouldn't be instantiated on its own - the parent class still contains abstract methods

Reply:
Ok, I saw it a bit out of context probably. 👍